install/image-references

```diff
@@ -0,0 +1,8 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: machine-api-operator
+    from:
+      kind: DockerImage
+      name: quay.io/coreos/machine-api-operator:b6a04c2
```

Comment:
this will need to point here now https://hub.docker.com/r/openshift/origin-machine-api-operator/tags/

Reply:
same as above